docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -426,14 +501,18 @@ Standard ONNX\* operators:
 | Cosh | No |
 | Crop | No |
 | CumSum | No |
+| DepthToSpace | No |
 | DequantizeLinear | No |
 | DetectionOutput (Intel experimental) | No |
 | Div | No |
 | Dropout | Not needed for inference |
 | Elu | No |
 | Equal | No |
 | Erf | No |
+| Exp | No |
 | Expand | No |
+| ExperimentalDetectronGroupNorm | No |
```

Comment:
Intel Experimental op.

Reply:
Actually, there are more `ExperimentalDetectron*` ops. Let's include all of them here with a comment that they are Intel experimental.